includes/Modules/Search_Console.php

```diff
@@ -423,54 +406,6 @@ protected function create_search_analytics_data_request( array $args = array() )
 			->query( $property_id, $request );
 	}
 
-	/**
-	 * Checks whether Search Console data exists for the given URL.
-	 *
-	 * The result of this query is stored in a transient.
-	 *
-	 * @since 1.0.0
-	 *
-	 * @param string $url The url to check data for.
-	 * @return bool True if Search Console data exists, false otherwise.
-	 */
-	protected function has_data_for_url( $url ) {
-		if ( ! $url ) {
-			return false;
-		}
-
-		$transient_key = 'googlesitekit_sc_data_' . md5( $url );
-		$has_data      = get_transient( $transient_key );
-
-		if ( false === $has_data ) {
-			/* @var Google_Service_Webmasters_ApiDataRow[]|WP_Error $response_rows Array of data rows. */
```

Comment:
Same here, imports for `Google_Service_Webmasters_ApiDataRow` and `Google_Service_Webmasters_SearchAnalyticsQueryResponse` are no longer used and can be removed.